xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -10,11 +10,14 @@
 
 #include "ServiceBroker.h"
 #include "cores/VideoPlayer/Interface/TimingConstants.h"
+#include "cores/VideoPlayer/VideoRenderers/OverlayRenderer.h"
```

Comment:
All the new includes are a bad sign. The Renderer calls the libass handler you shouldn't include the renderer here.

Settings are global for subtitles/text and define the user configured subtitle style, they are not specific to libass. I think this component should be self-contained: styles, resolution, etc should be propagated to this component not created by itself

Reply:
done